gen/definitions/logging.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: Logging
+path: Cisco-IOS-XR-um-logging-cfg:/logging
```

Comment:
Small cosmetic thing, we can remove the first '/' from the path and change this to: `Cisco-IOS-XR-um-logging-cfg:logging`.